Alexey/Text/guestbook/main/guestbook.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+def main():
```

Comment:
In task definition 
`It can allow comments or not and timestamps for all entries.`
so also I expect that you are able to read comments not all of them but by selected one

Reply:
Simply: add more functionality here
We can discuss it on QA sessions